src/controllers/export.ts

```diff
@@ -5,27 +5,32 @@
 
 import authentication from '@feathersjs/authentication';
 import cookieParser from 'cookie-parser';
+import db from '../models';
+
+import QueryStream from 'pg-query-stream';
+import JSONStream from 'jsonstream'
 
 export default (app) => {
   app.post('/export',
     cookieParser(),
     authentication.express.authenticate('jwt'),
     (req, res, next) => {
-      app.service('/events/bulk')
-        .find({
-          query: {
-            type: req.body.type,
-            startDate: req.body.startDate,
-            endDate: req.body.endDate,
-          },
-          user: (req as any).user,
-        })
-        .then((result) => {
-          res.setHeader('Content-Disposition', `attachment; filename=${req.body.type}-${req.body.startDate}.json`);
-          res.setHeader('Content-Type', 'application/json');
-          res.send(result);
+      db.sequelize.connectionManager.getConnection().then((pgConn) => {
+        const query = new QueryStream('SELECT * FROM events WHERE type = $1 AND "createdAt" > $2 AND "createdAt" <= $3', [
```

Comment:
Unclear semantics with dates. If they're interpreted as midnight, this does not actually address #29. If this was tested, great.